app/templates/account_details.hbs

```diff
@@ -0,0 +1,115 @@
+<div class="row checkout_pages">
+  <div class="small-12 medium-9 large-9 columns">
+    <section class="main-section new-user-section ui login_page">
+      <nav class="tab-bar">
+        <section class="left-small">
+          {{fa-icon "angle-left"}} {{#link-to "authenticate" classNames="back"}}{{t "back"}}{{/link-to}}
+        </section>
+
+        <section class="middle tab-bar-section">
+          <h1 class="title">{{t "account.account"}}</h1>
+        </section>
+      </nav>
+
+      {{#if model.order.length}}
```

Comment:
Would it be better to check cart rather than draft order?? User's draft Order could be empty also and it'll stil show this

Reply:
When draft order has no packages it gets deleted.